Modules/xSharePoint/DSCResources/MSFT_xSPVisioServiceApp/MSFT_xSPVisioServiceApp.schema.mof

```diff
@@ -0,0 +1,18 @@
+/*
+**Description**
+
+This resource is responsible for creating Visio Graphics Service Application instances within the local SharePoint farm.
+The resource will provision and configure the Visio Graphics Service Application.
+**Example**
+    xSPVisioServiceApp VisioServices
+    {
+        Name = "Visio Graphics Service Application"
+        ApplicationPool = "SharePoint Web Services"
+    }
+*/
+[ClassVersion("1.0.0.0"), FriendlyName("xSPVisioServiceApp")]
+class MSFT_xSPWebApplication : OMI_BaseResource
```

Comment:
The class name doesn't match - its copied from xSPWebApplication - should be MSFT_xSPVisioServiceApp